README.adoc

```diff
@@ -1507,7 +1507,7 @@ Favor `comp` over anonymous functions for function composition.
 
 === `partial` [[partial]]
 
-Favor `partial` over anonymous functions for currying.
+Favor `partial` over anonymous functions for partial application.
```

Comment:
It's seems to me it'd be best to just drop "for currying", as it's implied that `partial` does partial application.

Reply:
OK, I dropped it.